linux/setup/test_setup.yml

```diff
@@ -38,6 +38,10 @@
   include_tasks: ../utils/get_linux_system_info.yml
   when: guest_os_system_info_retrieved is undefined or not guest_os_system_info_retrieved
 
+- name: "Take a base snapshot if it does not exist"
```

Comment:
Why move taking base snapshot above here, VMware Tools status is not checked already running.

Reply:
line 25 gets vmtools status. `get_guest_ovt_version_build.yml` will call `get_installed_packages.yml` to check whether ovt package is installed. If Photon OS VM doesn't have python3-rpm installed, `get_installed_packages.yml` will fail due to lacking of dependency. So move `create_base_snapshot.yml` before `get_guest_ovt_version_build.yml` to force Photon OS to install all required packages, no matter whether it's installed or not.